wordpress-mobile/WordPress-iOS
# Large Titles / White Headers: Move My Sites VCs to My Sites Coordinator

This PR is the first part of pulling the changes from #15751 into develop. We uncovered some issues in that branch, so I'm moving things over piece by piece so we can test each in isolation. This PR implements the changes moving the main My Sites view controllers over to My Sites Coordinator. This will allow us to selectively switch between the new and old style blog pickers in an upcoming PR, from a purely Swift-based class.

**To test**

- Test on both iPad and iPhone.
- Build and run.
- Ensure that you can log in and out and back into the app and the blog details screens show up as expected.
- Try moving the iPad into multitasking to collapse the split view and back out and ensure that a detail view controller is restored in My Site.
- After logging out and back in, use the Memory Graph debugging view to ensure that there is only a single instance of WPTabBarController, 2 WPSplitViewControllers, and 1 BlogListViewController and BlogDetailViewController.
- Try performing an action that automatically navigates into a blog detail section. For example, from the home screen, tap and hold on the WordPress icon and choose Stats, or run a deep link using a command such as: `xcrun simctl openurl booted https://wordpress.com/comments/your_site_domain_here`

PR submission checklist:
